### ISSUE TITLE
Replacing CleanroomLogger

### ISSUE THREAD

**Issue by adrianniebla** [acheronfail/pixel-picker#30]
Is there a possibility of replacing CleanroomLogger? It appears it's missing the `arm64` architecture and won't compile on the M1 Macbooks.

**Comment by acheronfail** [comment]
That's quite unfortunate. Do you know of any other loggers which offer similar functionality and also support `arm64`?

**Comment by adrianniebla** [comment]
I don't but can work on it on my free time and test on Intel and ARM64 laptops.

**Comment by acheronfail** [comment]
That would be great, my access to macOS machines is limited at the moment.

For the time being I've commented out and removed the CleanroomLogger dependency completely on the branch `tmp/rm-cleanroomlogger`. And I've also created a release for it too - that should hopefully be enough for the time being: https://github.com/acheronfail/pixel-picker/releases/tag/1.4.1.

But we do need to either wait for CleanroomLogger to be compatible, or find a replacement. :)
